### PR TITLE
Bootstrap: Task Completed

### PR DESCRIPTION
**Pull request by anangkf**
What I do:
1. Summary
- [x] Added summary about bootstrap [d1be6da]
- [x] Repush summary after accidentally push [cf5427b]

2. Task
- [x] Added image assets [8ed3c76]
- [x] Added task: Navbar Header [fb35275]
- [x] Added task: Body [cbc50cc]
- [x] Added task: Footer [32b3f5b]
- [x] Added preview screenshots [5e7e87e]

### CODE REVIEW

**Review by syukronarie — COMMENTED** [review]
Summary: reviewed and approve

Tugasnya mana?

**Review by syukronarie — APPROVED** [review]

**Comment by anangkf** [comment]
masih belum kelar mas, ada kendala. untuk deadline tugas hari ini atau besok mas?

File: 07_Bootstrap/praktikum/index.html [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+        <!--  -->
+        <div class="d-flex mt-5 justify-content-center">
+            <div class="col-3 mx-3 d-flex flex-column align-items-center">
+                <div class="col-2 d-flex flex-column">
+                    <img src="assets/img/svg/cloud.svg" alt="">
+                </div>
+                <h6 class="text-uppercase py-4 fw-bold">introducing whatsapp</h6>
+                <p class="text-secondary">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Iure, ab!</p>
+                <p class="fw-semibold">Read More</p>
+            </div>
```
Comment (syukronarie) [inline 07_Bootstrap/praktikum/index.html:81]: kalo pengen pake symantic html untuk box image bisa pake figure > img > figcaption

reference:
[HTML `<figcaption>` Tag - w3school](https://www.w3schools.com/tags/tag_figure.asp)
[`<figcaption>`: The Figure Caption element - mdn web docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figcaption)

**Review by syukronarie — APPROVED** [review]
reviewed and approved. tks

**Comment by anangkf** [comment]
siap mas sarannya diterima

Pada tanggal Jum, 2 Sep 2022 23.28, M Arie Syukron ***@***.***>
menulis:

**Pull request merged**